src/main/java/org/apache/mesos/hdfs/config/ConfigServer.java

```diff
@@ -24,49 +27,69 @@
 import java.util.Set;
 import java.util.TreeSet;
 
+/**
+ * Provides configuration information through http API service
+ */
 public class ConfigServer {
 
+  public final Log log = LogFactory.getLog(ConfigServer.class);
+
   private Server server;
   private Engine engine;
-  private SchedulerConf schedulerConf;
+  private HdfsFrameworkConfig hdfsConf;
   private PersistentState persistentState;
 
   @Inject
-  public ConfigServer(SchedulerConf schedulerConf) throws Exception {
-    this(schedulerConf, new PersistentState(schedulerConf));
+  public ConfigServer(HdfsFrameworkConfig hdfsConf) {
+    this(hdfsConf, new PersistentState(hdfsConf));
   }
 
-  public ConfigServer(SchedulerConf schedulerConf, PersistentState persistentState)
-      throws Exception {
-    this.schedulerConf = schedulerConf;
+  public ConfigServer(HdfsFrameworkConfig hdfsConf, PersistentState persistentState) {
+    this.hdfsConf = hdfsConf;
     this.persistentState = persistentState;
     engine = new Engine();
-    server = new Server(schedulerConf.getConfigServerPort());
+    server = new Server(hdfsConf.getConfigServerPort());
     ResourceHandler resourceHandler = new ResourceHandler();
-    resourceHandler.setResourceBase(schedulerConf.getExecutorPath());
+    resourceHandler.setResourceBase(hdfsConf.getExecutorPath());
     HandlerList handlers = new HandlerList();
     handlers.setHandlers(new Handler[]{
         resourceHandler, new ServeHdfsConfigHandler()});
     server.setHandler(handlers);
-    server.start();
+
+    try {
+      server.start();
+
+      // NOPMD jetty throws a generic exception, we have to catch it!
+    } catch (Exception e) {
+      final String msg = "unable to start jetty server";
+      log.error(msg, e);
+      throw new ConfigServerException(msg);
+    }
   }
 
-  public void stop() throws Exception {
-    server.stop();
+  public void stop() {
+    try {
+      server.stop();
+    } catch (Exception e) {
```

Comment:
You're just going to silently drop this exception? Why not throw a ConfigServerException here too?